rootpw-allow-ssh.ks.in

```diff
@@ -0,0 +1,41 @@
+#test name: rootpw-allow-ssh
+#
+# Test that root ssh login with password is working.
+
+# Use defaults except for users and common (no shutdown)
+%ksappend repos/default.ks
+%ksappend l10n/default.ks
+%ksappend network/default.ks
+%ksappend storage/default.ks
+%ksappend payload/default_packages.ks
+
+# allow ssh to root
+rootpw --plaintext --allow-ssh qweqwe
+
+# Reboot the installed system.
+reboot
+
+# Validate on the first boot.
+%ksappend validation/success_on_first_boot.ks
+
+# Set up the actual test.
+%post
+
+# Write the code to run after reboot.
+cat > /usr/libexec/kickstart-test.sh << 'EOF'
+
+dnf install -y sshpass > /dev/null
```

Comment:
Are the BaseOS/AppStream repositories somehow enabled on the installed rhel-9 system? I'm asking because they are not enabled by default there.

Reply:
Good point. It might be better to install `sshpass` during the installation.